dev/tasks/tasks.yml

```diff
@@ -431,7 +431,7 @@ tasks:
 {############################## Wheel OSX ####################################}
 
 # enable S3 support from macOS 10.13 so we don't need to bundle curl, crypt and ssl
-{% for macos_version, macos_codename, arrow_s3 in [("10.9", "mavericks", "OFF"),
+{% for macos_version, macos_codename, arrow_s3_gcs in [("10.9", "mavericks", "OFF"),
```

Comment:
I have a strong opinion for this but I like adding a new variable for `gcs` for readability:

```text
{% for macos_version, macos_codename, arrow_s3, arrow_gcs in [("10.9", "mavericks", "OFF", "OFF"),
                                                              ("10.13", "high-sierra", "ON", "ON")] %}
```

Reply:
done.

Reply:
Thanks!

(I had a typo: "I have a strong opinion" -> "I don't have a strong opinion")